lib/api/apiUtils/bucket/bucketCreation.js

```diff
@@ -165,20 +165,21 @@ function bucketLevelServerSideEncryption(bucket, headers, log, cb) {
  */
 function createBucket(authInfo, bucketName, headers,
     locationConstraint, log, cb) {
+    // Prefer using undefined instead of null for unused properties so they are not present in metadata payload
```

Comment:
Why this comment ? Maybe make more sense to put that in that class `BucketInfo` ? And/or add a check in the constructor to transform null to undefined ?

Reply:
I would say I agree with Mickael here, but as we change how the function is called at the same time, it can be a bit risky. We could have some logic relying explicitly on "null", but here, best is to trust the tests, if they pass we are good.

So better to mention, in the `BucketInfo` class, that we should not pass null but undefined to get the default values?

Foir the `null` I agree to keep it if provided, and not fallback to `undefined`.